backend/server/websocket_manager.py

```
@@ -92,24 +93,25 @@ async def disconnect(self, websocket: WebSocket):
             # Still try to close the connection if possible
             try:
                 await websocket.close()
-            except:
+            except Exception:
                 pass  # If this fails too, there's nothing more we can do
 
-    async def start_streaming(self, task, report_type, report_source, source_urls, document_urls, tone, websocket, headers=None, query_domains=[], mcp_enabled=False, mcp_strategy="fast", mcp_configs=[]):
+    async def start_streaming(self, task, report_type, report_source, source_urls, document_urls, tone, websocket, headers=None, query_domains=[], mcp_enabled=False, mcp_strategy="fast", mcp_configs=[], max_search_results=None):
```

Ruff (0.15.5)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)